lib/taste_tester/ssh_util.rb

```diff
@@ -0,0 +1,127 @@
+module TasteTester
+  class SSH
+    module Util
+      def ssh_base_cmd
+        jumps = TasteTester::Config.jumps ?
+          "-J #{TasteTester::Config.jumps}" : ''
+        "#{TasteTester::Config.ssh_command} #{jumps} -T -o BatchMode=yes " +
+          '-o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no ' +
+          "-o ConnectTimeout=#{TasteTester::Config.ssh_connect_timeout} " +
+          "#{TasteTester::Config.user}@#{@host} "
+      end
+
+      def error!
+        error = <<~ERRORMESSAGE
+  SSH returned error while connecting to #{TasteTester::Config.user}@#{@host}
+  The host might be broken or your SSH access is not working properly
+  Try doing
+
+      #{ssh_base_cmd} -v
+
+  to see if ssh connection is good.
+  If ssh works, add '-v' key to taste-tester to see the list of commands it's
+  trying to execute, and try to run them manually on destination host
+        ERRORMESSAGE
+        logger.error(error)
+        fail TasteTester::Exceptions::SshError
+      end
+
+      def build_ssh_cmd(ssh, command_list)
+        if TasteTester::Config.windows_target
+          # Powershell has no `&&`. So originally we looked into joining the
```

Comment:
yeah I played with that a bit but we also want to catch the "testing by someone else" differently, and I found that I was losing that... I'm sure it can be made ti work...